doc/samtools-fixmate.1

```diff
@@ -82,6 +82,9 @@ is selected.
 .TP
 .BI --no-PG
 Do not add a @PG line to the header of the output file.
+.TP 8
+.B --index-format STR
```

Comment:
Looks like the wrong lines were pasted in here.  It's part of 72230203 which added all the `-@, --threads` stuff.